code/src/java/plugin/dicebag/DiceBagPlugin.java

```diff
@@ -141,13 +122,13 @@ public void stop()
     @Override
 	public int getPriority()
 	{
-		return SettingsHandler.getGMGenOption(LOG_NAME + ".LoadOrder", 20);
+		return SettingsHandler.getGMGenOption(DiceBagPlugin.LOG_NAME + ".LoadOrder", 20);
```

Comment:
This change is because it is referencing a static field from a non-static context. I like this stylistically and it makes it easier to move code around.
My formatter is detecting whatever the current file is using I think. I didn't touch the @overide so it remains a weird mix.